lib/beaker-pe/install/pe_utils.rb

```diff
@@ -702,9 +704,20 @@ def setup_defaults_and_config_helper_on(host, master, acceptable_exit_codes=nil)
         #Install PE based on global hosts with global options
         #@see #install_pe_on
         def install_pe
+          options.merge!(pe_presets)
+
+          if options[:disable_analytics] == true then 
+            disable_analytics(dashboard)
+          end
+
           install_pe_on(hosts, options)
         end
 
+        def disable_analytics(hosts)
+          logger.info("Disabling analytics on dashboard host(s)")
+          on(hosts, 'echo "127.0.0.1\tgoogle-analytics.com\n" >> /etc/hosts')
```

Comment:
you should checkout beaker's `set_etc_hosts` method and see if that can do this for you, because it sets these host entries in a platform-independent manner.